plugin/js/EpubPacker.js

```diff
@@ -131,7 +131,7 @@ EpubPacker.prototype = {
     addManifestItem: function(manifest, href, id, mediaType) {
         let that = this;
         var item = that.createAndAppendChild(manifest, "item");
-        item.setAttribute("href", href);
+        item.setAttribute("href", href.substr(6));
```

Comment:
In certain places (specifically the manifest only) having "OEBPS/" before each item causes the list index to be out of range because of where content.opf and toc.ncx is.

We remove "OEBPS/" Only when inserting chapter href's into the manifest and nowhere else. Doing this anywhere else breaks a lot of things.